R/utils.R

```diff
@@ -115,7 +115,14 @@ get.available.mran.date <- function(date0, date1) {
 
   # Report mid value
   n.dates <- length(available.dates)
-  mid.date.k <- round(n.dates / 2, 0)
+
+  if (n.dates == 0) {
+    message1("You required a package version that was not archived on MRAN")
```

Comment:
@urisohn, feel free to edit this new error message as you see fit

Reply:
Maybe "We looked for the version of the package you need in MRAN but it was not found there"